source/client/output_formatter_impl.h

```diff
@@ -49,6 +49,15 @@ class YamlOutputFormatterImpl : public OutputFormatterImpl {
   absl::StatusOr<std::string> formatProto(const nighthawk::client::Output& output) const override;
 };
 
+class CsvOutputFormatterImpl : public OutputFormatterImpl {
```

Comment:
I note that this wasn't followed here before, but could we add a class comment as per the style guide?

https://google.github.io/styleguide/cppguide.html#Class_Comments